mlos_core/mlos_core/optimizers/README.md

```diff
@@ -0,0 +1,18 @@
+This is a directory that contains wrappers for different optimizers to integrate into MLOS.
+This is implemented though child classes for the `BaseOptimizer` class defined in `optimizer.py`.
+
+The main goal of these optimizers is to take a suggest configurations based on prior samples to find an optimum based on some objective. This process is interacted with through and ask and tell interface.
```

Comment:
```suggestion
The main goal of these optimizers is to `suggest` configurations, possibly based on prior trial data to find an optimum based on some objective(s).
This process is interacted with through `register` and `suggest` interfaces.
```

Fixed some language.

Also, one sentence per line please.  Easier to review diffs later.